taskset_daemon.sh

```diff
@@ -0,0 +1,6 @@
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+script=python $DIR/gpu_monitor.py -v -t
+
+cmd="watch -n 300 "
+$cmd "$script"
+
```

Comment:
I would not include this script into the repository. If you want, you can add a section at the end of the README, e.g. `Periodically Setting CPU Affinities` and put those 4 lines there, maybe explain a little bit.